subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -1496,27 +1492,123 @@ static isoal_status_t isoal_tx_try_emit_pdu(struct isoal_source *source,
 	return err;
 }
 
+/**
+ * @brief Get the next unframed payload number for transmission based on the
+ *        input meta data in the TX SDU and the current production information.
+ * @param  source_hdl[in]      Destination source handle
+ * @param  tx_sdu[in]          SDU with meta data information
+ * @param  payload_number[out] Next payload number
+ * @return                     Number of skipped SDUs
+ */
+uint16_t isoal_tx_unframed_get_next_playload_number(isoal_source_handle_t source_hdl,
```

Comment:
```suggestion
uint16_t isoal_tx_unframed_get_next_payload_number(isoal_source_handle_t source_hdl,
```

Reply:
Thanks. Amazing that that got through all the reviews.

Reply:
Done.